packages/router/src/__tests__/util.test.ts

```diff
@@ -121,12 +132,45 @@ describe('matchPath', () => {
   })
 
   it('transforms a param for Globs', () => {
-    expect(
-      matchPath('/version/{globbyMcGlob...}', '/version/path/to/file')
-    ).toEqual({
+    //single
+    expect(matchPath('/version/{path...}', '/version/path/to/file')).toEqual({
+      match: true,
+      params: {
+        path: 'path/to/file',
+      },
+    })
+
+    //  multiple
+    expect(matchPath('/a/{a...}/b/{b...}/c', '/a/1/2/b/3/4/c')).toEqual({
+      match: true,
+      params: {
+        a: '1/2',
+        b: '3/4',
+      },
+    })
+
+    // adjacent
+    expect(matchPath('/a/{a...}{b...}/c', '/a/1/2/3/4/c')).toEqual({
```

Comment:
Did you leave out the slash between `{a...}` and `{b...}` on purpose?
If you did, what would be the expected behavior *with* the slash?

Reply:
Yes, I wanted to test what happens if you DO happen to have two globs next to each other. It's a fine question about what would happen if there is a slash in there, I will try it out!

Reply:
Added a test with a slash between. The first glob matches as much as it can and the 2nd grabs the last bit, which is as expected.

Reply:
Oh interesting edge case.  Makes me curious about globs that start and end in slashes as well.  You've likely already thought about it and fiddled with it... but if not I'm curious if that has any impact.